warplib/keccak.cairo

```diff
@@ -1,126 +1,110 @@
-%lang starknet
-
-from starkware.cairo.common.alloc import alloc
-from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
-from starkware.cairo.common.cairo_keccak.keccak import keccak_bigend
-from starkware.cairo.common.dict_access import DictAccess
-from starkware.cairo.common.math_cmp import is_le_felt
-from starkware.cairo.common.uint256 import Uint256
-from starkware.cairo.common.bitwise import bitwise_and
-
-from warplib.maths.pow2 import pow2
-from warplib.maths.utils import get_min, narrow_safe
-from warplib.memory import wm_to_felt_array
-
-const BYTES_IN_FELT = 8;
-const BITS_IN_BYTE = 8;
-
-func warp_keccak{
-    range_check_ptr, bitwise_ptr: BitwiseBuiltin*, warp_memory: DictAccess*, keccak_ptr: felt*
-}(loc: felt) -> (output: Uint256) {
-    alloc_locals;
-    let (input_len, input) = wm_to_felt_array(loc);
-    let (packed_bytes_len, packed_bytes) = pack_bytes_felt(BYTES_IN_FELT, 0, input_len, input);
+use array::ArrayTrait;
+use memory::wm_to_felt_array;
 
-    let (res: Uint256) = keccak_bigend{keccak_ptr=keccak_ptr}(packed_bytes, input_len);
+const BYTES_IN_FELT: felt = 8;
+const BITS_IN_BYTE: felt = 8;
 
-    return (res,);
+fn warp_keccak(loc: felt) -> u256 {
+    let (input_len, input) = wm_to_felt_array(loc);
+    let (packed_bytes_len, packed_bytes) = pack_bytes_felt(BYTES_IN_FELT, 0, input_len, input);
+    // return statement
+    return keccak_bigend(packed_bytes, input_len);
```

Comment:
I understand the function is not implemented yet? Then it should be marked with `TODO: add import once it's available in corelib`